src/components/treeView/createEntityNode.tsx

```diff
@@ -113,7 +116,8 @@ export class CreateEntityNode extends React.Component<CreateEntityNodeProps, Cre
 				renderStrategy = this.props.inputRenderStrategy(this.state.nameInputValue, this.handleEnterInput, this.onInputChange, this.setInputRefAndFocus);
 				break;
 			case 'CreateError':
-				renderStrategy = this.props.createErrorRenderStrategy(this.state.nameInputValue, this.onInputChange);
+				// TODO (machiam) Don't use JSON.stringify; parse out API error instead, or map it to a localized error
```

Comment:
Are we planning on doing this?

Reply:
It seems pretty essential.

Reply:
Yep we are, I think it's much easier to do it while we're implementing this at the integration level.